Big-Spender/readme.md

```diff
@@ -68,7 +69,13 @@ INSERT YOUR QUERY HERE
 **You:** Then here's the query for that:
 
 ```sql
-INSERT YOUR QUERY HERE
+INSERT YOUR QUERY here
+select * from spends where lower(description) like '%fees%'; 
+
+OR
+
+select * from spends where description iLIKE '%fees%'; 
```

Comment:
These are two separate queries, right? 

Nice use of both lower and `ilike `, but what about if I want` fee`? Will this this appear? How can we modify this to include both fees and fee?